### DISCUSSION TITLE
Immediately start saving Trace Log to file

### DISCUSSION THREAD

**Discussion by TFWol** [sandboxie-plus/Sandboxie#2487] [Q&A Feedback] [answered]
I have a program that crashes Sandboxie, as soon as it's run, preventing me from saving the trace log.

How do I make the trace save to a file in realtime as soon as it's enabled?
I couldn't find a procedure when I searched.

**Comment by isaak654** [comment, marked as answer]
Currently, you can only enable the Trace Log tab and manually click the [Save to file](https://user-images.githubusercontent.com/12372772/203755642-0f6aae52-5d21-43ae-9440-899a11d20b5b.png) icon when you're done.

Please note that there are a couple of crash reports about existing logging features:  #2466 #2471

For your case, I would recommend to open a [new issue](https://github.com/sandboxie-plus/Sandboxie/issues/new/choose) and attach one or more crash dumps related to the malfunctioning program.

![crash_dump_how_to](https://user-images.githubusercontent.com/12372772/203767833-fa38c899-8c46-47c4-9965-d0fde1bad3e5.png)

[Generate a kernel or complete crash dump](https://learn.microsoft.com/en-us/troubleshoot/windows-client/performance/generate-a-kernel-or-complete-crash-dump)

**Reply by TFWol** [reply]
Thanks for the answer. I thought I was overlooking a tracing feature while troubleshooting and thought I'd ask before diving down the issues path.

Probably should have mentioned `EnableMiniDump=y` option won't generate anything for this particular crash and no .dmp files I could find.
I guess it's more like a freeze.

I'll look at trying to generate an actual crash to get a dmp file to spawn

**Reply by isaak654** [reply]
Ok, then please open a new [problem report](https://github.com/sandboxie-plus/Sandboxie/issues/new/choose) by filling in as many details as possible.
